src/ClientGenerator/CSharpCodeGenerator.cs

```diff
@@ -160,7 +162,7 @@ protected override void AddCreateObjectReferenceMethods(GrainInterfaceData grain
             var invokerField = new CodeSnippetTypeMember(fieldImpl);
             factoryClass.Members.Add(invokerField);
 
-            var methodImpl = String.Format(@"
```

Comment:
Whichever way we go will involve a "huge number" of changes. :(

In the whole codebase [under `src` excluding any codegen code] there are 465 uses of `String.Format` and 326 uses of `string.Format`.

Item 11 in the CoreFx C# coding style says use keyword types (ie `string`) instead of BCL types (eg `String`) and that is the rule that ReSharper is applying here.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md